spec/spec.yaml

```diff
@@ -77,6 +77,11 @@ tags:
       A release provides _components_, like for example Kubernetes. For each
       release the contained components are listed. Changes in components are
       detailed in the _changelog_ of a release.
+  - name: cluster labels
+    description: |
+      Cluster labels allow users to group and tag clusters with own key value pairs.
+
+      Note: Cluster labeling is currently (as of February 2020) only available on <span class="badge aws">AWS</span> to selected Giant Swarm customers.
```

Comment:
Why this note?

Reply:
Following the discussion in giantswarm/api#979 and [this comment](https://github.com/giantswarm/giantswarm/issues/1693#issuecomment-578764717) I decided to add this for now only for v5 clusters

Reply:
I understand - I would be happy if we at least bring it to other providers as well.

Reply:
But okay to sort AWS first.

Reply:
Yeah but now as you wrote:
> But ok to sort AWS first
I think this notice should be rewritten to something like

`Cluster labeling is currently (as of February 2020) only available on cluster releases 10 or higher on <span class="badge aws">AWS</span>.`